Source/ChocolateyGui.Common/Models/AppConfiguration.cs

```diff
@@ -96,6 +96,10 @@ public class AppConfiguration
         [Feature]
         public bool? PreventUsageOfUpdateAllButton { get; set; }
 
+        [LocalizedDescription(nameof(Resources.SettingsView_ToggleAllowGlobalConfirmationDescription))]
+        [Feature]
+        public bool? AllowGlobalConfirmation { get; set; }

```

Comment:
I'm honestly not sure about calling it `AllowGlobalConfirmation` would be the best.

This implies that there are other confirmations that will be made as well.
And (although I know it is used in Chocolatey CLI), a feature called `AllowGlobalConfirmation` doesn't say much of what it does.

Would it maybe be better with something like:

`IgnoreGlobalDialogConfirmation` or `SkipGlobalDialogConfirmation` (Maybe even without the Global Mention?)

Thoughts @gep13

Reply:
Part of me likes the consistency with using the `AllowGlobalConfirmation` so that is matches what is in Chocolatey itself, however, I tend to agree with @AdmiringWorm here.

Naming is hard, but of the options provided, I would lean towards `SkipGlobalDialogConfirmation` or perhaps `SkipModalDialogConfirmation`.

Reply:
IMO, `SkipModalDialogConfirmation` sounds like the best option here.

Reply:
I like it, I've updated it to be SkipModalDialogConfirmation.